cmd/poseidon/poseidon.go

```diff
@@ -121,11 +121,14 @@ func main() {
 		panic(err)
 	}
 	defer conn.Close()
-	// Check if frimament grpc service is available and then proceed
+	// Check if firmament grpc service is available and then proceed
 	WaitForFirmamentSerive(fc)
 	go schedule(fc)
 	go stats.StartgRPCStatsServer(statsServerAddress, firmamentAddress)
 	kubeVer := strings.Split(kubeVersion, ".")
+	if len(kubeVer) < 2 {
+		glog.Fatalf("Incorrect content in --kubeVersion %s, kubeVersion should be in the format of X.Y", kubeVersion)
+	}
 	kubeMajorVer, err := strconv.Atoi(kubeVer[0])
 	if err != nil {
 		glog.Fatalf("Incorrect content in --kubeVersion %s", kubeVersion)
```

Comment:
BTW, can you add detailed message here and below as well?

Reply:
Sure， Done